packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -2400,7 +2400,11 @@ class SelectWithFooter extends React.Component {
           aria-labelledby={titleId}
           isDisabled={isDisabled}
           direction={direction}
-          footer={<Button variant="link" isInline>Action</Button>}
+          footer={ <>
+            <Button variant="link" isInline>
+              Action
+            </Button>
+            </>}
```

Comment:
Curious about the empty tag, I recall this did not work with PF docs. Has it been enabled?

That said, an empty tag doesn't seem necessary here.

Reply:
Correct.  I was testing with two buttons and the empty tag was a left over. 
I am not apply to reproduce the issue with skipping the "Miss"